.changeset/swift-ties-rest.md

```diff
@@ -0,0 +1,21 @@
+---
+'@backstage/plugin-bazaar-backend': minor
+---
+
+**BREAKING** Changes to add authorization for the PUT, POST and DELETE /projects endpoints. The bazaar plugin now requires a `PermissionEvaluator' is passed to the router.
```

Comment:
I don't love this as a breaking change, could we make it optional and add a 404 check to the function you've written?

Also,
```suggestion
**BREAKING** Changes to add authorization for the PUT, POST and DELETE /projects endpoints. The bazaar plugin now requires a `PermissionEvaluator` is passed to the router.
```

Reply:
Yes, I will make this change optional. 
Where would you like me to do a 404 check? In `evaluateRequestPermission`?